pages/api/behavior.ts

```diff
@@ -27,32 +23,15 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
 
 }
 
-const getBehaviors = async (req: NextApiRequest, res: NextApiResponse) => {
-  try {
-    const behaviors = await prisma.behavior.findMany();
-    return res.status(200).json(behaviors); 
-  } catch (error) {
-    return res.status(500).json({ error });
-  }
-}
-
-const getBehavior = async (req: NextApiRequest, res: NextApiResponse) => {
-  const { id } = req.query;
-  try {
-    const behavior = await prisma.behavior.findUnique({
-      where: { id: Number(id) }
-    });
-    return res.status(200).json(behavior);
-  } catch (error) {
-    return res.status(500).json({ error });
-  }
-}
-
 const createBehavior = async (req: NextApiRequest, res: NextApiResponse) => {
   try {
     const behaviorData: Behavior = req.body;
     const behavior = await prisma.behavior.create({
-      data: behaviorData
+      data: {
```

Comment:
Here we are getting an error because we don't include some of the properties of Behavior.

For example, `trialsPerEntry` is nullable, but it will complain at us unless we do the following `trialsPerEntry: null`. This also goes for `entries` and `tags`, although these two are not nullable.

For now, I would initialize these properties like so on Behavior creation:
` entries: [], trialsPerEntry: null, tags: []`

@pariahGH can you comment on this?

Reply:
May need to double check that the library was regenerated fromk the schema - nullable we should be able to just not pass in the property at all. It should be yelling at us foor having `null` in entries/tags, as `[]` modifier prevents nullability - [here](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#-modifier)